### ISSUE TITLE
My Ledger wallet seems to always crash. I get the following error

### ISSUE THREAD

**Issue by drinkyd** [monero-project/monero#6740]
Hi, 

With the GUI it never finishes syncing and crashes. 

With the CLI, I get to "Starting refresh..." and get this error:

terminate called after throwing an instance of 'std""runtime_error'
 what(); wrong Device Status: 0x6f42 (UNKNOWN), EXPECTED 0x9000 (SW_OK), MASK 0xffff

Does anyone know what is causing this and how I can sync my wallet to the local chain I downloaded without crashing? I'm running the latest version of Monero on window 10 with SSD drive. 

Thanks.

**Comment by selsta** [comment]
Are you running the latest Ledger firmware and Ledger monero app and monero software?

Can you try to recreate your wallet?

**Comment by drinkyd** [comment]
Yes latest version of everything. When you say recreate wallet? Do you mean to reset my ledger?

**Comment by selsta** [comment]
> Do you mean to reset my ledger? 

No, just create a new monero wallet with your Ledger.

Also can you try to use a different computer?

**Comment by drinkyd** [comment]
If I create a new wallet, how do I transfer my funds to this new wallet if I can't access the funds due to the crashing?

**Comment by selsta** [comment]
If you restore your Ledger wallet on a different computer your funds will be there, no transfer necessary.

**Comment by drinkyd** [comment]
Yes I tried it on another computer. And now the GUI just closes automatically during syncing. *sigh* I think I'm going to give up. It shouldn't be that difficult accessing funds.  Any more ideas what I can do? I've been dealing with this for months.

**Comment by selsta** [comment]
So both CLI and GUI also have issues on a different computer?

You can follow this tutorial to convert your Ledger seed into a Monero seed: 
https://monero.stackexchange.com/questions/11979/how-to-convert-ledger-seed-to-monero-compatible-seed-on-windows

This should only be done as a last step if everything else fails as it exposes your Ledger seed to your computer.
